config-site/llnl_lc.cmake

```diff
@@ -0,0 +1,80 @@
+#      the distribution.
+#    * Neither the name of the LLNS/LLNL nor the names of its
+#      contributors may be used to endorse or promote products derived
+#      from this software without specific prior written permission.
+# 
+# THIS SOFTWARE  IS PROVIDED BY  THE COPYRIGHT HOLDERS  AND CONTRIBUTORS
+# "AS  IS" AND  ANY EXPRESS  OR IMPLIED  WARRANTIES, INCLUDING,  BUT NOT
+# LIMITED TO, THE IMPLIED  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A  PARTICULAR  PURPOSE ARE  DISCLAIMED.  IN  NO  EVENT SHALL  LAWRENCE
+# LIVERMORE  NATIONAL SECURITY, LLC,  THE U.S.  DEPARTMENT OF  ENERGY OR
+# CONTRIBUTORS BE LIABLE FOR  ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR  CONSEQUENTIAL DAMAGES  (INCLUDING, BUT NOT  LIMITED TO,
+# PROCUREMENT OF  SUBSTITUTE GOODS  OR SERVICES; LOSS  OF USE,  DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER  IN CONTRACT, STRICT LIABILITY,  OR TORT (INCLUDING
+# NEGLIGENCE OR  OTHERWISE) ARISING IN  ANY WAY OUT  OF THE USE  OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# 
+# This work was produced at Lawrence Livermore National Laboratory under
+# Contract No.  DE-AC52-07NA27344 with the DOE.
+# 
+# Neither the  United States Government nor  Lawrence Livermore National
+# Security, LLC nor any of  their employees, makes any warranty, express
+# or  implied,  or  assumes  any  liability or  responsibility  for  the
+# accuracy, completeness,  or usefulness of  any information, apparatus,
+# product, or  process disclosed, or  represents that its use  would not
+# infringe privately-owned rights.
+# 
+# Any reference herein to  any specific commercial products, process, or
+# services by trade name,  trademark, manufacturer or otherwise does not
+# necessarily  constitute or imply  its endorsement,  recommendation, or
+# favoring  by  the  United  States  Government  or  Lawrence  Livermore
+# National Security,  LLC. The views  and opinions of  authors expressed
+# herein do not necessarily state  or reflect those of the United States
+# Government or Lawrence Livermore National Security, LLC, and shall not
+# be used for advertising or product endorsement purposes.
+#
+##############################################################################
+
+
+###
+# Default settings for lc systems
+###
+
+###
+# Helpers for TP lib location
+###
+set(SILO_HOME /usr/gapps/silo)
+# Want to use the newer compiler, right? So append the correct one to the SYS_TYPE
+set(SILO_ARCH $ENV{SYS_TYPE}_gcc.6.1.0)
+
+##
+# HDF5
+##
+silo_option_default(VAR SILO_HDF5_DIR VALUE ${SILO_HOME}/hdf5/1.8.16/${SILO_ARCH})
+
+##
+# SZIP
+##
+silo_option_default(VAR SILO_SZIP_DIR VALUE ${SILO_HOME}/szip/2.1/${SILO_ARCH})
+
+##
+# QT5 (must be the location of Qt5Config.cmake)
+##
+silo_option_default(VAR SILO_QT5_DIR VALUE /usr/WS1/visit/visit/thirdparty_shared/3.2.0/toss3/qt/5.14.2/linux-x86_64_gcc-6.1/lib/cmake/Qt5)
```

Comment:
I have some build recipies I just used to build and install 4.11 on LC OCF and SCF systems in a shell script there if those would be helpful.

Reply:
Not sure what you mean. I thought the config-site's were something created to aid future builds on those hosts.

I was just trying to follow your  config-site plans, having lc-common entries put in a lc config-site, linux-common in a linuc config-site, then  host specific including the common entries, does that make sense.
I like it better than VisIt's way where every host config-site duplicates so  much that is common .